soem/ethercatfoe.c

```diff
@@ -57,8 +57,6 @@
 #include "ethercatmain.h"
 #include "ethercatfoe.h"
 
-#define EC_MAXFOEDATA 512
```

Comment:
We'll commit the change in ethercatfoe with my comment, the changes in firmware we'll leave out.